### PR TITLE
Rename `about.txt` to `plugin.txt` and update information within it

### PR DESCRIPTION
**Pull request by bene-dictator**
~~Waiting on https://github.com/endless-sky/endless-sky/pull/9496~~

That PR has been merged.

### CODE REVIEW

File: plugin.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name "Enter the name of your plugin here. This will be displayed in the list of plugins when you install it."
+about "Enter some text that describes your plugin here. Endless Sky will then display it when the user has your plugin installed and selected."
+about "Use multiple 'about' nodes to create new paragraphs in the description box, if necessary."
```
Comment (quyykk) [inline plugin.txt:3]: Using multiple `about` nodes doesn't actually work yet :sweat_smile:

Comment (bene-dictator) [reply plugin.txt:3]: Oh, I thought I saw that somewhere : )

**Comment by quyykk** [comment]
@bene-dictator I think we should do the multiple `about` thingy for the next release and then merge this when we release it.

**Comment by bene-dictator** [comment]
@Amazinite is this waiting on anything now that endless-sky/endless-sky#9496 is merged?

**Comment by Amazinite** [comment]
I didn't even know this was a thing.

**Review by Amazinite — APPROVED** [review]

**Pull request merged**